runtime/cmake/caches/aix_ppc-64.cmake

```diff
@@ -21,18 +21,20 @@
 ################################################################################
 
 #TODO: Env vars should be auto detected by platform
+
+# Prevent CMake from automatically creating export lists for shared libraries
+set(CMAKE_XL_CreateExportList "" CACHE INTERNAL "")
+
 set(J9VM_ARCH_POWER ON CACHE BOOL "")
 set(J9VM_ENV_DATA64 ON CACHE BOOL "")
 set(J9VM_ENV_DLPAR ON CACHE BOOL "")
 set(J9VM_ENV_HAS_FPU ON CACHE BOOL "")
 set(J9VM_ENV_SHARED_LIBS_CALLEE_GLOBAL_TABLE_SETUP OFF CACHE BOOL "")
 set(J9VM_ENV_SHARED_LIBS_USE_GLOBAL_TABLE ON CACHE BOOL "")
 
-# Prevent CMake from automatically creating export lists for shared libraries
-set(CMAKE_XL_CreateExportList "" CACHE INTERNAL "")
-
-set(J9VM_GC_TLH_PREFETCH_FTA OFF CACHE BOOL "")
+set(J9VM_GC_SPARSE_HEAP_ALLOCATION ON CACHE BOOL "")
```

Comment:
Is the plan to change all the references in JIT code from `J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION` to `J9VM_GC_SPARSE_HEAP_ALLOCATION`?

Reply:
Yes. @rmnattas, apologies for the inconvenience, but please update the JIT references.

Reply:
Is it also in the plan to update UMA builds to support this flag?

Reply:
Yes, about to merge that PR.